conf.pri.mingw

```diff
@@ -0,0 +1,58 @@
+# Adapt these paths
+
+# Point this to the boost include folder
```

Comment:
Not related to this PR: I find these comments (Point this to ...) somewhat redundant.